src/app/core/notifications/notifications_manager.nim

```diff
@@ -13,6 +13,7 @@ logScope:
 const NOTIFICATION_SOUND = "qrc:/imports/assets/audio/notification.wav"
 
 # Signals which may be emitted by this class:
+const SIGNAL_ADD_NOTIFICATION_TO_ACTIVITY_CENTER* = "addNotificationToActivityCenter"
```

Comment:
According to [new message notification diagram](https://drive.google.com/file/d/1L_9c2CMObcDcSuhVUu97s9-_26gtutES/view) and [other actions diagram](https://drive.google.com/file/d/1KmG7lJDJIx6R_HJWeFvMYT2wk32RoTJQ/view) we should add notification to the Activity Center. Because of that `SIGNAL_ADD_NOTIFICATION_TO_ACTIVITY_CENTER` is introduced here and it's emitted whenever we need to add new notification to the Activity Center (so in theory we just need to connect to it in the activity center module and that's it). 

But that part is not done here, because that would add notifications correctly, according to the logic described in diagrams, but only while the app is running, after app restart we won't have them since `status-go` part doesn't know about those notifications (that way they may be added only on the UI side). 

So we need to discuss here what and how should we handle this? 
1. Whether to update `status-go` in a way that the desktop app calls some method from the `status-go` to update DB in order to have the same list of notifications in the activity center after app restart (in this case the app will be source of truth and determine whether to display a notification or not, and if yes, use `status-go` endpoint just to store it in the DB)? 
2. Or maybe to apply the same logic, as we currently have in `NotificationsManager` class, on the side of `status-go` and let `status-go` notifies the app about activity center notifications.

Reply:
> I think we should do number 2.

@jrainville yes, that's correct and that's the plan, but since this is a big change, a logic which follows [message notification diagram](https://drive.google.com/file/d/1L_9c2CMObcDcSuhVUu97s9-_26gtutES/view) is firstly applied here.

Also at this moment `SIGNAL_ADD_NOTIFICATION_TO_ACTIVITY_CENTER` signal is not used anywhere, it is added more just to mark the single place in the code when and where from we should add notification to the Activity Center.

We need at least 2 steps more for this ticket:
1. move notification settings which are now stored in the local .ini file to the settings on the status-go side, we already have a ticket for that https://github.com/status-im/status-desktop/issues/5649
2. move logic from the desktop app to the `status-go` that items added to the Activity Center will be properly stored and consistent after app restart and also that the same logic may be shared between mobile and desktop app